frontend/src/components/teacher/view-sessions/TestSessionListItemPopover.tsx

```diff
@@ -29,19 +30,19 @@ const TestSessionListItemPopover = ({
   const history = useHistory();
 
   const [deleteTestSession] = useMutation(DELETE_TEST_SESSION, {
-    variables: { id: testSessionEditingData.testSessionId },
+    variables: { id: session.testSessionId },
     refetchQueries: [getQueryName(GET_TEST_SESSIONS_BY_TEACHER_ID)],
   });
 
   const onEditTestSession = () => {
-    history.push(Routes.DISTRIBUTE_ASSESSMENT_PAGE, testSessionEditingData);
+    history.push(Routes.DISTRIBUTE_ASSESSMENT_PAGE, session);
```

Comment:
if we're sending the whole session object, can we update this [line](https://github.com/uwblueprint/jump-math/blob/a874401de960ad06629997b6677bc12ea2bdfd73/frontend/src/components/pages/teacher/DistributeAssessmentPage.tsx#L25)? also in that case we can delete the `TestSessionEditingData` type.

Reply:
sure